tests/integration_tests/base_api_tests.py

```diff
@@ -288,6 +288,30 @@ def test_get_filter_related_owners(self):
         # TODO Check me
         assert expected_results == sorted_results
 
+    def test_get_base_filter_related_owners(self):
+        """
+        API: Test get base filter related owners
+        """
+        self.login(username="admin")
+        uri = f"api/v1/{self.resource_name}/related/owners"
+        self.app.config["EXCLUDE_USER_USERNAMES"] = ["gamma"]
+        gamma_user = (
+            db.session.query(security_manager.user_model)
+            .filter(security_manager.user_model.username == "gamma")
+            .one_or_none()
+        )
+        assert gamma_user is not None
+        users = db.session.query(security_manager.user_model).all()
+
+        rv = self.client.get(uri)
+        assert rv.status_code == 200
+        response = json.loads(rv.data.decode("utf-8"))
+        assert response["count"] == len(users) - 1
+        response_users = [result["text"] for result in response["result"]]
+        assert "gamma user" not in response_users
+        # revert the config change
+        self.app.config["EXCLUDE_USER_USERNAMES"] = []
```

Comment:
nit: we should cache the previous config value and restore to cache instead of empty list

Reply:
yes, implemented a `with_config` decorator per @villebro sugestion